proofs/hol_light/arm/proofs/mlkem_poly_basemul_acc_montgomery_cached_k2.ml

```diff
@@ -378,6 +378,7 @@ let poly_basemul_acc_montgomery_cached_k2_SPEC = prove(poly_basemul_acc_montgome
     CONV_TAC INT_RING
 );;
 
+(* NOTE: This needs to be kept in sync with the CBMC spec in native/api.h *)
```

Comment:
Should be `mlkem/native/aarch64/src/arith_native_aarch64.h`?

Reply:
Yes